packages/containers/src/ComponentForm/kit/defaultRegistry.js

```diff
@@ -111,10 +111,59 @@ function onError({ errors, error, schema }) {
 	return { errors: getNewErrors(errors, schema, extractErrorMessage(error)) };
 }
 
+/**
+ * Update the content of the properties
+ * @param body is the new value
+ * @param trigger the trigger that call it
+ * @example
+const trigger = {
+	"action":"guessMe",
+	"family":"test",
+	"options":[
+		{
+			"path":"root.updatable_config",
+			"type":"object" // or "string"
+		}
+	],
+	"parameters":[
+		{
+			"key":"arg0.name",
+			"path":"root.updatable_config.name"
+		}
+	],
+	"type":"update"
+};
+ */
+function updateProperties({ body, trigger, properties }) {
+	const newProperties = { ...properties };
+	const targetPath = trigger.options[0].path;
+	const getValue = () => {
+		if (trigger.options[0].type === 'object') {
+			return body;
+		}
+		return body.data;
+	};
+	const targetSplitted = targetPath.split('.');
+	targetSplitted.reduce((acc, current, index) => {
+		if (!acc[current]) {
+			// eslint-disable-next-line no-param-reassign
+			acc[current] = {};
```

Comment:
it's really a good thing to do that? #ByPassESLint

Reply:
In the context of a `reduce()` call it's not that shocking, and regarding eslint automatically flagging it, it is/has been debated on the eslint repo itself https://github.com/eslint/eslint/issues/8581
A workaround for that would be to return the deconstructed/reconstructed accumulator with values update

Reply:
not shocking but yet easy to avoid